src/pages/Groups/Group3/styles/Screen1.styled.ts

```diff
@@ -0,0 +1,45 @@
+import styled from 'styled-components'
+
+export const MainContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  margin: 5%;
```

Comment:
is this intentional? makes top nav have a spacing that is responsive....
![Screenshot 2021-10-10 at 11 36 47 PM](https://user-images.githubusercontent.com/57870728/136702835-ade77575-8d71-4bbb-949b-d6a9847d55f7.png
![Screenshot 2021-10-10 at 11 37 42 PM](https://user-images.githubusercontent.com/57870728/136702871-3a48d17f-e869-43d5-b4bf-5e66670e13b2.png)
)

Reply:
resolved in commit b63baef